test/Microsoft.AspNet.Mvc.FunctionalTests/Logging/LoggingTests.cs

```diff
@@ -0,0 +1,199 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if ASPNET50
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using LoggingWebSite;
+using LoggingWebSite.Controllers;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Mvc.Logging;
+using Microsoft.AspNet.TestHost;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.FunctionalTests
+{
+    public class LoggingTests
+    {
+        private readonly IServiceProvider _services = TestHelper.CreateServices(nameof(LoggingWebSite));
+        private readonly Action<IApplicationBuilder> _app = new LoggingWebSite.Startup().Configure;
+
+        [Fact]
+        public async Task AssemblyValues_LoggedAtStartup()
```

Comment:
Feel free to separate out these tests into different files by category if it simplifies your life. 

```
LoggingStartupTest.cs
LoggingActionSelectionTest.cs